files/README.md

```diff
@@ -1,6 +1,5 @@
 ## Standard Workflow
 
```

Comment:
In the Standard Workflow, all of the items are numbered as 1.  Does this need to be descending order?

Reply:
That's one way to write an ordered list in Markdown. It will be ascending but it already was previously so I left it that way.